### DISCUSSION TITLE
AGW Installation

### DISCUSSION THREAD

**Discussion by robinge330** [magma/magma#6182] [Q&A]
https://docs.magmacore.org/docs/lte/setup_deb
 The set-up is based on above link.
currently on journalctl -f we see below info.
Please could you guide any debug traces to understand why the installation is not progressing.
 appreciate the guidance. 

 journalctl -f 
Apr 16 15:15:40 mgm-agw systemd[681]: Startup finished in 21ms.
Apr 16 15:15:40 mgm-agw systemd[1]: Started User Manager for UID 0.
Apr 16 15:15:40 mgm-agw systemd[1]: Started Session 1 of user root.
Apr 16 15:16:06 mgm-agw sshd[699]: Accepted password for root port 3514 ssh2
Apr 16 15:16:06 mgm-agw sshd[699]: pam_unix(sshd:session): session opened for user root by (uid=0)
Apr 16 15:16:06 mgm-agw systemd[1]: Started Session 3 of user root.
Apr 16 15:16:06 mgm-agw systemd-logind[644]: New session 3 of user root.
Apr 16 15:17:01 mgm-agw CRON[728]: pam_unix(cron:session): session opened for user root by (uid=0)
Apr 16 15:17:01 mgm-agw CRON[729]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Apr 16 15:17:01 mgm-agw CRON[728]: pam_unix(cron:session): session closed for user root
Apr 16 15:17:47 mgm-agw systemd[1]: Starting Cleanup of Temporary Directories...
Apr 16 15:17:47 mgm-agw systemd[1]: systemd-tmpfiles-clean.service: Succeeded.
Apr 16 15:17:47 mgm-agw systemd[1]: Started Cleanup of Temporary Directories.

**Comment by robinge330** [comment]
Resolved by commenting the reboot and maintained inly one reboot at the end.
This script will need to be updated
cd ~/magma/lte/gateway/deploy
./agw_install.sh

**Reply by amintgan** [reply]
Hello @robinge330 please how do I check my current AGW version that was installed in 2019 and how do I upgrade to the recent release
